### PR TITLE
Restrict pyvmomi dependency to 7.0.3

### PR DESCRIPTION
**Pull request by kunal-pmj**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
       packages=[],
       install_requires=[
         'lxml >= 4.3.0',
-        'pyVmomi >= 6.7',
+        'pyVmomi <=7.0.3, >=6.7',
```
Comment (karaatanassov) [inline setup.py:16]: Is there something compatiblity issue with Pyvmomi 8? It just shipped.

https://github.com/vmware/pyvmomi/releases

Comment (kunal-pmj) [reply setup.py:16]: Yes, https://github.com/vmware/pyvmomi/issues/993

Comment (karaatanassov) [reply setup.py:16]: Got it. We should se how to enable somehow 8.0.0.1.patch1 whenever it comes. I se a fix is in place already but no new release.

**Comment by aagrawal3** [comment]
There are 2 more files with similar changes. requirements.py and requirements_pypi.py 
Please make the change there as well.

**Pull request merged**